test/performance/scripts/load.js

```diff
@@ -0,0 +1,48 @@
+import http from "k6/http";
+import { check } from "k6";
+import { generateBundle } from "../resources/ips-bundle-transaction.js";
+
+const BASE_URL = __ENV.BASE_URL || "http://localhost:5001";
+
+export const options = {
+  scenarios: {
+    load: {
+      executor: "constant-arrival-rate",
+      rate: 50,
+      timeUnit: "1s",
+      duration: "5m",
+      preAllocatedVUs: 50,
+      maxVUs: 200,
+    },
```

Comment:
I think we should adjust the test params to give us passing results with the platform in it's current state.